save_data.rb

```diff
@@ -0,0 +1,29 @@
+module SaveAll
+  def save_book(books)
+    book_to_save = []
+    books.each do |book|
+      book_to_save << { 'title' => book.title, 'author' => book.author }
+    end
+    File.write('./books.json', JSON.generate(book_to_save))
+  end
+
+  def save_people(people)
+    people_to_save = []
+    people.each do |person|
+      people_to_save << { 'name' => person.name,
+                          'age' => person.age,
+                          'id' => person.id }
+    end
+    File.write('./persons.json', JSON.generate(people_to_save))
+  end
+
+  def save_rentals(rentals)
+    rentals_to_save = []
+    rentals.each do |rental|
+      rentals_to_save << { 'date' => rental.date,
+                           'person_name' => rental.person.name,
+                           'book_title' => rental.book.title, 'person_id' => rental.person.id }
+    end
```

Comment:
- [ ] The  'date' in your save_rentals is not defined it returns an error while saving the rentals.

![Capture d’écran 2022-08-31 103604](https://user-images.githubusercontent.com/61562626/187648741-a3f3ee30-b791-4a5b-9815-b46bfb8ee6a8.png)